Tests.ReactView/Tests.ReactView.csproj

```diff
@@ -16,9 +16,11 @@
 
   <ItemGroup>
     <PackageReference Include="Avalonia.Desktop" Version="0.10.6" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.1" />
```

Comment:
why introducing new refs?

Reply:
To avoid transitive references, this way we are making them direct references and are forcing their version into one with the vulnerability fix

Reply:
do you understand that these vulnerabilities are not important, right? (react view is a closed environment) even more... this impacts a testing project... so its not important at all.

Reply:
yes I do and like you said being this a closed environment and a test project is almost impossible for anyone to explore them, however, we compromise ourselves to reduce the number of identified High vulnerabilities.

Reply:
or you accept (and ignore) these vulnerabilities because they are not real vulnerabilities and invest your time in real important things :)

Reply:
I invested 2 min doing the fix, Im ok with it :D